docs/pages/guides/analyzing-bundles.mdx

```diff
@@ -5,49 +5,64 @@ description: Learn about improving the production JavaScript bundle size of Expo
 
 ### Using Atlas with `npx expo start`
 
-You can use Expo Atlas with the local development server. This method allows Atlas to update whenever you change any code in your project. 
+You can use Expo Atlas with the local development server. This method allows Atlas to update whenever you change any code in your project.
 
 Once your app is running using the local development server on Android, iOS, and/or web, you can open Atlas through the [dev tools plugin menu](/debugging/devtools-plugins/#using-a-dev-tools-plugin) using <kbd>shift</kbd> + <kbd>m</kbd>.
 
-<Terminal cmd={[
-  '# Start the local development server with Atlas',
-  '$ EXPO_UNSTABLE_ATLAS=true npx expo start',
-]} />
+<Terminal
+  cmd={[
+    '# Start the local development server with Atlas',
+    '$ EXPO_UNSTABLE_ATLAS=true npx expo start',
+  ]}
+/>
 
 #### Changing development mode to production
 
 By default, Expo starts the local development server in [development mode](/workflow/development-mode/#development-mode). Development mode disables some optimizations that are enabled in [production mode](/workflow/development-mode/#production-mode). You can also start the local development server in production mode to get a more accurate representation of the production bundle size:
 
-<Terminal cmd={[
-  '# Run the local development server in production mode',
-  '$ EXPO_UNSTABLE_ATLAS=true npx expo start --no-dev', 
-]} />
+<Terminal
+  cmd={[
+    '# Run the local development server in production mode',
+    '$ EXPO_UNSTABLE_ATLAS=true npx expo start --no-dev',
+  ]}
+/>
 
 ### Using Atlas with `npx expo export`
 
 You can also use Expo Atlas when generating a production bundle for your app or EAS Update. Atlas generates a **.expo/atlas.jsonl** file during export, which you can share and open without having access to the project.
 
-<Terminal cmd={[
-  '# Export your app for all platforms',
-  '$ EXPO_UNSTABLE_ATLAS=true npx expo export',
-  '',
-  '# Open the generated Atlas file',
-  '$ npx expo-atlas .expo/atlas.jsonl',
-]} />
+<Terminal
+  cmd={[
+    '# Export your app for all platforms',
+    '$ EXPO_UNSTABLE_ATLAS=true npx expo export',
+    '',
+    '# Open the generated Atlas file',
+    '$ npx expo-atlas .expo/atlas.jsonl',
+  ]}
+/>
 
 You can also specify the platforms you want to analyze using the `--platform` option. Atlas will gather the data for the exported platforms only.
 
+### Analyzing transformed modules
+
+<ContentSpotlight alt="Expo Atlas module page." src="/static/images/atlas/atlas-module.avif" />
+
+Inside Atlas, you can command + click on a graph node to see the transformed module details. This feature helps you understand how a module is transformed by Babel, which modules it imports, and which modules imported it. This can be used to trace the origin of a module across the dependency graph.
```

Comment:
```suggestion
Inside Atlas, you can press <kbd>⌘ Cmd</kbd> and click on a graph node to see the transformed module details. This feature helps you understand how a module is transformed by Babel, which modules it imports, and which modules imported it. This can be used to trace the origin of a module across the dependency graph.
```

Formatting nit.